### PR TITLE
bug in index names and reading from configuration

### PR DESCRIPTION
**Pull request by gserranojc**
Error when create two indexes with the same name for two distinct tables.

### CODE REVIEW

File: sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala [hunk 1]
```diff
@@ -46,8 +46,8 @@ class ValidatingPropertyMap[K, V](val m: Map[K, V]) {
 
   def getString(key: K, default: Option[String]): Option[String] = {
```
Comment (arinconstrio) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: There are 20 calls to this method, only one of them has a default value, mostly other has None as second param..., we need to point to the "def getString(key: K): String" these calls

**Comment by arinconstrio** [comment]
I didn't found the Cassandra index name length limit, we need to take care about it

File: sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala (hunk 1 above)
Comment (gserranojc) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: We need Option[String] in result, "def getString(key: K): String" is not possible.
We can add a new function "def getString(key: K): Option[String]" ???

Comment (danielcsant) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: Where do we need that function?
Could we use properties.getOption or just properties.get?

Comment (gserranojc) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: We have getOption now. But I think that the function that control the "" value is better and necessary.

Comment (danielcsant) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: It is just used in ElasticSearchOutput right? In 
`override val defaultIndexMapping = properties.getString("indexMapping", Some(DEFAULT_INDEX_TYPE))`

defaultIndexMapping always is going to have a value so there is no point to treat it as an Option

Here we could use
`override val defaultIndexMapping = properties.getString("indexMapping", DEFAULT_INDEX_TYPE)`

Comment (gserranojc) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: Ok Dani, we must change this code in ElasticSearch Output, we should add this issue.

**Pull request merged**